internal/controller/policymanifest_controller.go

```diff
@@ -93,7 +93,7 @@ func (r *PolicyManifestReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// Set kyvernoPolicyException destination namespace.
 	kyvernoPolicyException.Namespace = r.DestinationNamespace
 	// Set kyvernoPolicyException name.
-	kyvernoPolicyException.Name = fmt.Sprintf("gs-kpo-%s-exceptions", polman.ObjectMeta.Name)
+	kyvernoPolicyException.Name = fmt.Sprintf("gs-kpo-%s", polman.ObjectMeta.Name)
```

Comment:
This is fine with me. Just `kpo` would also be sufficient. Is the name meaningful anywhere (I assume/hope not)? If not, the prefix could even be configurable.